spec/msf/core/auxiliary/auth_brute_spec.rb

```diff
@@ -46,6 +46,13 @@ def create_tempfile(content)
         expect { |block| subject.each_user_pass(true, &block) }.not_to yield_control
       end
     end
+
+    it 'calculates the size correctly' do
+      subject.each_user_pass(true) do |_user, _pass|
+        # noop
+      end
+      expect(subject.class.class_variable_get("@@max_per_service")).to eq(options[:expected_size])
```

Comment:
Reaching into the state like this isn't great, but I'm intentionally leaving the implementation untouched